site/gatsby-site/src/utils/date.js

```diff
@@ -22,3 +24,11 @@ export const isPastDate = {
     return parsedDate <= today;
   },
 };
+
+export const formatDate = (dateString) => {
+  const date = new Date(dateString);
+
+  return `${date.getFullYear()}-${String(date.getMonth() + 1).padStart(2, '0')}-${String(
+    date.getDate()
+  ).padStart(2, '0')}`;
+};
```

Comment:
Do we need this custom Date format function? Maybe we can use the same that we currently using:
`import { format } from 'date-fns';`